src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIUserGroupsService.java

```diff
@@ -157,15 +157,16 @@ public void updateUserGroupDetails(Long groupId, FieldUpdate update) {
 	 * Add a new member to the user group
 	 *
 	 * @param groupId identifier for the {@link UserGroup}
-	 * @param request details about the user to add to the user group
+	 * @param userId identifier for the {@link User}
+	 * @param role   role to assign to the user
 	 * @param locale  current users {@link Locale}
 	 * @return message to the user about the status of this request
 	 */
-	public String addMemberToUserGroup(Long groupId, NewMemberRequest request, Locale locale) {
+	public String addMemberToUserGroup(Long groupId, Long userId, String role, Locale locale) {
```

Comment:
I think this should probably be annotated with `@Transactional` since adding a user to a group would mean that the group and associations would have to be checked.

Reply:
Added in c427089ae2f1a86e46a20ce76e32f226b2ca3b89